src/components/structures/MatrixChat.tsx

```diff
@@ -84,6 +84,14 @@ import DialPadModal from "../views/voip/DialPadModal";
 import { showToast as showMobileGuideToast } from '../../toasts/MobileGuideToast';
 import SpaceStore from "../../stores/SpaceStore";
 import SpaceRoomDirectory from "./SpaceRoomDirectory";
+import {
```

Comment:
Is this import required?

Reply:
No, I had tons of code to be cleaned up , I must have missed it. I apologize, this will be removed.

Reply:
Please refer to pull request #12